content/2_Bronze/Complete_Rec.mdx

```diff
@@ -115,15 +124,20 @@ n = int(input())
 weights = list(map(int, input().split()))
 
 
-def solve(i, sum1, sum2):
+def recurse_apples(i: int, sum1: int, sum2: int) -> int:
+	# We've added all apples- return the absolute difference
```

Comment:
```suggestion
	# We've added all apples - return the absolute difference
```

Reply:
does this really matter? if you insist i'll change it